javascripts/software-search.js

```diff
@@ -0,0 +1,108 @@
+const getSoftware = async () => {
```

Comment:
Without some sort of transpile step, the ES2015+ syntax won't run in Internet Explorer. Are we okay to limit support to newer browsers?

![image](https://user-images.githubusercontent.com/1779930/89801388-18275280-dafe-11ea-99be-f741064c6888.png)

Aside from syntax, `fetch` and promises would also not be supported by default without pulling in some polyfill ([`@babel/polyfill`](https://babeljs.io/docs/en/babel-polyfill), [`core-js`](https://github.com/zloirock/core-js)).

Reply:
Yeah, considered that. Since the target user base is TTS, I don't think it's worth the effort to support IE.